packages/vm/src/testing/mock.rs

```diff
@@ -9,15 +9,15 @@ use cosmwasm_std::{
 use super::storage::MockStorage;
 use crate::{Api, Extern, FfiResult, Querier, QuerierResult};
 
-static CONTRACT_ADDR: &str = "cosmos2contract";
+pub const MOCK_CONTRACT_ADDR: &str = "cosmos2contract";
```

Comment:
Nice to expose this... I've been cut/pasting it into test cases, which is not ideal.

Reply:
Yeah, same here.